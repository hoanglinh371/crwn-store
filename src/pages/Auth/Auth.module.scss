.container {
  display: flex;
  justify-content: space-between;
  width: 1200px;
  margin: 32px auto;
}

@media screen and (max-width: 600px) {
  .container {
    max-width: 100%;
    flex-direction: column;
    gap: 48px;
  }
}
