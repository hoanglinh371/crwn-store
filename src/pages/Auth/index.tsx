import React from 'react';
import classNames from 'classnames/bind';

import styles from './Auth.module.scss';

import SignIn from 'src/features/auth/components/SignIn';
import SignUp from 'src/features/auth/components/SignUp';

const cx = classNames.bind(styles);

const Auth: React.FC = () => {
    return (
        <div className={cx('container')}>
            <SignIn />
            <SignUp />
        </div>
    );
};

export default Auth;
