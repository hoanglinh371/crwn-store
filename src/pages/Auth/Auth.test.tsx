import { cleanup, render } from '@testing-library/react';

import Auth from '.';

describe('Auth component', () => {
  afterEach(cleanup);

  it('Should_MatchSnapshot', () => {
    const { container } = render(<Auth />);
    expect(container).toMatchSnapshot();
  });
});
