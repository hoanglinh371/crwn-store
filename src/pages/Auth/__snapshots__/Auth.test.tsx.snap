// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth component Should_MatchSnapshot 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="container"
    >
      <h2>
        Already have an account?
      </h2>
      <span>
        Sign in with your email and password
      </span>
      <form>
        <div
          class="group"
        >
          <input
            class="form-input"
            id=":r0:"
            name="email"
            type="email"
            value=""
          />
          <label
            class="form-input-label"
            for=":r0:"
          >
            Email
          </label>
        </div>
        <div
          class="group"
        >
          <input
            class="form-input"
            id=":r1:"
            name="password"
            type="password"
            value=""
          />
          <label
            class="form-input-label"
            for=":r1:"
          >
            Password
          </label>
        </div>
        <div
          class="button-container"
        >
          <button
            aria-label="button"
            class="container"
            type="submit"
          >
            Sign In
          </button>
          <button
            aria-label="button"
            class="container google"
            type="button"
          >
            Google Sign In
          </button>
        </div>
      </form>
    </div>
    <div
      class="container"
    >
      <h2>
        Don't have an account?
      </h2>
      <span>
        Sign up with your email and password
      </span>
      <form>
        <div
          class="group"
        >
          <input
            class="form-input"
            id=":r2:"
            name="displayName"
            type="text"
            value=""
          />
          <label
            class="form-input-label"
            for=":r2:"
          >
            Display Name
          </label>
        </div>
        <div
          class="group"
        >
          <input
            class="form-input"
            id=":r3:"
            name="email"
            type="email"
            value=""
          />
          <label
            class="form-input-label"
            for=":r3:"
          >
            Email
          </label>
        </div>
        <div
          class="group"
        >
          <input
            class="form-input"
            id=":r4:"
            name="password"
            type="password"
            value=""
          />
          <label
            class="form-input-label"
            for=":r4:"
          >
            Password
          </label>
        </div>
        <div
          class="group"
        >
          <input
            class="form-input"
            id=":r5:"
            name="confirmPassword"
            type="password"
            value=""
          />
          <label
            class="form-input-label"
            for=":r5:"
          >
            Confirm Password
          </label>
        </div>
        <button
          aria-label="button"
          class="container"
          type="submit"
        >
          Sign Up
        </button>
      </form>
    </div>
  </div>
</div>
`;
